src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5047,4 +5047,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RuntimeDoesNotSupportDefaultInterfaceImplementationForMember" xml:space="preserve">
     <value>'{0}' cannot implement interface member '{1}' in type '{2}' because the target runtime doesn't support default interface implementation.</value>
   </data>
+  <data name="ERR_DefaultInterfaceImplementationModifier" xml:space="preserve">
+    <value>The modifier '{0}' is not valid for this item in C# {1}. Please use language version {2} or greater.</value>
```

Comment:
I am following established pattern used by several similar error messages. For example:
```
  <data name="ERR_FeatureNotAvailableInVersion7" xml:space="preserve">
    <value>Feature '{0}' is not available in C# 7. Please use language version {1} or greater.</value>
  </data>
```